R/package.R

```diff
@@ -176,4 +176,18 @@ initialize_python <- function(required_module = NULL, use_environment = NULL) {
 }
 
 
-
+should_configure <- function() {
+  
+  # allow users to opt out
+  configure_ok <- Sys.getenv("RETICULATE_AUTOCONFIGURE", unset = "TRUE")
+  configure_ok <- if (configure_ok %in% c("FALSE", "False", "0")) FALSE else TRUE
```

Comment:
Could just use an early return false here

Reply:
right, guess it's programming not logic :-)
updated